config/build/manager/manager.yaml

```diff
@@ -23,11 +23,9 @@ spec:
         control-plane: controller-manager
     spec:
       containers:
-      - command:
-        - /manager
-        args:
+      - args:
         - --enable-leader-election
-        image: controller:latest
+        image: github.com/projectriff/system/cmd/managers/build
         name: manager
```

Comment:
Also, would be nice to move name and/or image as the top level field, making the list-of-maps easier to read

Reply:
I'm indifferent, but would accept a contribution.